stage1/usr_from_coreos/manifest.d/systemd.manifest

```diff
@@ -61,6 +61,9 @@ lib64/libmount.so.1
 lib64/libmount.so.1.1.0
 lib64/libnss_files-2.21.so
 lib64/libnss_files.so.2
+lib64/libpam.so
```

Comment:
silent tear.

Can we document it somewhere?

On Wed, Apr 27, 2016 at 2:07 PM, Iago López Galeiras <
notifications@github.com> wrote:

> In stage1/usr_from_coreos/manifest.d/systemd.manifest
> <https://github.com/coreos/rkt/pull/2514#discussion_r61245810>:
>
> > @@ -61,6 +61,9 @@ lib64/libmount.so.1
> >  lib64/libmount.so.1.1.0
> >  lib64/libnss_files-2.21.so
> >  lib64/libnss_files.so.2
> > +lib64/libpam.so
>
> Good old manual testing.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/rkt/pull/2514/files/8fed61d3c7b78d72433f8eb995995ffec51d08fe#r61245810>
>